projectile.el

```diff
@@ -1551,10 +1551,14 @@ Regular expressions can be used."
   "Check if FILE should be ignored.
 
 Regular expressions can be used."
-  (cl-some
-   (lambda (name)
-     (string-match-p name file))
-   (projectile-ignored-files)))
+   (or (cl-some
+        (lambda (suf)
+          (string-suffix-p suf file t))
+        projectile-globally-ignored-file-suffixes)
```

Comment:
Wasn't this supposed to be handled by `projectile-ignored-files`?

Reply:
It isn't, `projectile-ignored-files` only handles `projectile-globally-ignored-files`.

Reply:
I think we should just put the handling of the suffixes there as well. Otherwise it becomes a big weird to have external handling for this.

Reply:
I am not sure how to do this efficiently, because `projectile-ignored-files` merely lists entries the `.projectile` file and `projectile-globally-ignored-files` variable. If we want it to list all project files with extensions in `projectile-globally-ignored-file-suffixes` as well, we would first need to recompute the list of project files. But this seems overkill to me. Do you have any suggestion on how to proceed?

Reply:
Well, we can always expand the suffixes as `**/*.suffix` patterns in the context of the broader function. If this wildcard expansion doesn't work out well I guess we should go with your suggestion. 

Still, the changes will have to be covered by some tests.